combat-trainer.lic

```diff
@@ -291,6 +291,8 @@ class LootProcess
     @dump_junk = settings.dump_junk
     echo("  @dump_junk: #{@dump_junk}") if $debug_mode_ct
     @dump_timer = Time.now - 300
+    # remove || 9 after a few days when base.yaml migrates
+    @dump_item_count = settings.dump_item_count || 9
```

Comment:
Is it possible for people to get in a state where base.yaml isn't updated but the script is? I feel like it has happened before so I put in this safeguard to be on the safe side.

Reply:
👏

Reply:
It'd be super awesome if you could make the PR to remove this as soon as this PR is merged, with a note to let it sit for a few days. Otherwise I'll completely forget about this.